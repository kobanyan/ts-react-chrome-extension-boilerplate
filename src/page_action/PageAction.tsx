import * as React from 'react';

export const PageAction = () => {
  return (
    <div>
      <p>{chrome.i18n.getMessage('pageAction')}</p>
    </div>
  );
};
