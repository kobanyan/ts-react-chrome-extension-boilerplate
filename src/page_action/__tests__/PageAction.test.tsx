import * as React from 'react';
import { PageAction } from '../PageAction';
import * as renderer from 'react-test-renderer';

describe('Page Action', () => {
  it('should match snapshot', () => {
    const component = renderer.create(<PageAction />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
