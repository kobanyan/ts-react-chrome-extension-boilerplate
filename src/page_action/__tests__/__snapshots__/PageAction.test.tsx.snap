// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Action should match snapshot 1`] = `
<div>
  <p>
    This is a popup of page action!
  </p>
</div>
`;
